tasks/task1/task1_1.js

```diff
@@ -0,0 +1,8 @@
+process.stdin.setEncoding('utf8');
+
+process.stdin.on('readable', function() {
+    let chunk;
+    while ((chunk = process.stdin.read()) !== null) {
```

Comment:
Also, think of utilising a do..while loop to have an ability to read a chunk inside a loop.
E.x you can store an empty string in chunk before entering the loop and then at the end of it read another portion of data.

Reply:
Could you please explain more ? I'm little bit confused

Reply:
The main idea is to make the logic more straightforward.
Consider the following pseudo-code:

`dataHandler
    declare empty chunk variable
    start loop
        transform and output chunk
        read the next part
    end loop if chunk is null`